Atomic/atomic.py

```diff
@@ -435,7 +435,33 @@ def scan(self):
                 fetch = 1
             else:
                 fetch = 0
-            scan_return = json.loads(oscap_i.scan_list(scan_list, 4, fetch, timeout=99999))
+
+            json_result = ""
+            try:
+                token = oscap_i.CVEScanListAsync(
+                    scan_list, 4, fetch
+                )
+                try:
+                    while True:
+                        success, json_result = oscap_i.GetCVEScanListAsyncResults(token)
+                        if success:
+                            break
+                        time.sleep(1)
+                except:
+                    oscap_i.CancelCVEScanListAsync(token)
+                    raise
```

Comment:
Hi,
this "empty" raise reraises the exception that was caught. The only thing this does is issue a cancel command and then reraise so something else handles the real exception. The best reference I found for this are the Python docs - https://docs.python.org/2/tutorial/errors.html - search for "re-raise"